birdbox/microsite/models.py

```diff
@@ -140,6 +140,17 @@ class Meta:
         ),
     )
 
+    canonical_rel = CharField(
+        default="",
+        blank=True,
+        max_length=512,
+        help_text=mark_safe(
+            'Value to use within &lt;link rel="canonical" ...&gt; tag in the head of this page, to indicate a preferred URL for this page elewhere. '
```

Comment:
typo: 'elewhere'

Reply:
Instead of a `CharField` I was expecting a `URLField`. Was there a reason to opt for one over the other?

Reply:
Good point re URLField - will make the swap now